server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -224,6 +225,12 @@ while read size result ;do
     if [[ $status -ne 0 ]] ;then
         echo "$TS: prefix failure - code $status:         mkdir -p $RESULTS/$hostname/$prefix" >&4
     else
+	echo "ln -s $TMP_PATH/$PROG/$hostname/$resultname $incoming"
```

Comment:
Indentation.

This will end up in the logfile. I presume that's the intent? IOW, this is not debugging output, right?